From f3b54e4da3c6f8d864ff35a0da9c113b553ad4ca Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Sun, 23 Feb 2020 20:32:06 -0700
Subject: [PATCH 20/23] Revert "x86/boot/compressed: Remove unnecessary
 sections from bzImage"

Arvind's new series will fix this in a different way.

This reverts commit e11831d0ada3efc7f9268f35c80257f7e1b1dd0c.

Link: https://github.com/ClangBuiltLinux/linux/issues/899
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 arch/x86/boot/compressed/vmlinux.lds.S | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/x86/boot/compressed/vmlinux.lds.S b/arch/x86/boot/compressed/vmlinux.lds.S
index 12a20603d92e..508cfa6828c5 100644
--- a/arch/x86/boot/compressed/vmlinux.lds.S
+++ b/arch/x86/boot/compressed/vmlinux.lds.S
@@ -73,9 +73,4 @@ SECTIONS
 #endif
 	. = ALIGN(PAGE_SIZE);	/* keep ZO size page aligned */
 	_end = .;
-
-	/* Discard all remaining sections */
-	/DISCARD/ : {
-		*(*)
-	}
 }
-- 
2.25.1

